src/main/java/page_objects/BasePage.java

```diff
@@ -14,6 +14,8 @@
     private final By tabChangePassword = By.cssSelector("li a[href='/Account/ChangePassword.cshtml']");
     private final By tabLogout = By.cssSelector("li a[href='/Account/Logout']");
     private final By tabRegister = By.cssSelector("li a[href='/Account/Register.cshtml']");
+    private final By tabBookticket = By.cssSelector("li a[href='/Page/BookTicketPage.cshtml']");
+    private final By tabMyticket = By.cssSelector("li a[href='/Page/ManageTicket.cshtml']");
```

Comment:
![image](https://user-images.githubusercontent.com/23515968/132319901-7102aa94-42a7-4b93-8f69-29e2284f7717.png)

tabMyTicket is already existed
tabBookticket is wrong format

Reply:
I fixed it